docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2916 @@
+* Adds alert details page feature flag by App {kibana-pull}142839[#142839]
+* Adds new contextual attributes to Infrastructure - Inventory Rule {kibana-pull}140598[#140598]
+
+Platform::
+Adds notifications plugin, offering basic email service {kibana-pull}143303[#143303]
+
+Security::
+The API Keys screen can be accessed in a Read Only view with the the cluster privilege `read_security` {kibana-pull}144923[#144923]
+
+Uptime::
+Adds `created_at` field in saved objects {kibana-pull}143507[#143507]
+
+For more information about the features introduced in 8.6.0, refer to <<whats-new,What's new in 8.6>>.
+
+[[enhancements-and-bug-fixes-v8.6.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.6.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.6.0]]
+=== Enhancements
+Alerting::
+* Clone rule {kibana-pull}144741[#144741]
+* Remove errors and warning in triggers_actions_ui jest test {kibana-pull}144443[#144443]
+
+* Increase the default table size of the cases table to 10 {kibana-pull}144228[#144228]
+* Bulk enable rules api {kibana-pull}144216[#144216]
+* Create bulk delete on rules front {kibana-pull}144101[#144101]
+* Improve Task Manager’s retry logic for ad-hoc tasks {kibana-pull}143860[#143860]
+* Increases the max length limit of the case title to 160 characters {kibana-pull}143664[#143664]
+* Adds the ability to bulk edit tags in the cases table {kibana-pull}143450[#143450]
+* Filter cases without assignees {kibana-pull}143390[#143390]
+* Make actions retry when encountering failures {kibana-pull}143224[#143224]
+* Adds a backlink to cases when pushing in external services {kibana-pull}143174[#143174]
+* Move Connectors to own page {kibana-pull}142485[#142485]
+* 142183 create bulk delete on rules {kibana-pull}142466[#142466]
+* Allow `_source` field for ES DSL query rules {kibana-pull}142223[#142223]
+* Update rule status {kibana-pull}140882[#140882]
+
+APM::
+* Mobile most used pie charts with Lens {kibana-pull}144232[#144232]
+* Multi metric partition charts {kibana-pull}143966[#143966]
+* Adds missing options to APM central config {kibana-pull}143668[#143668]
+* Adds waterfall to dependency operations {kibana-pull}143257[#143257]
+* Display configuration table above code sample {kibana-pull}143178[#143178]
+* Implement improvements to AWS Lambda metrics view {kibana-pull}143113[#143113]
+* Display featured integrations at top of integrations list per category {kibana-pull}142771[#142771]
+
+Dashboard::
+* Adds unmapped runtime field support to options list {kibana-pull}144947[#144947]
+* Adds "Exists" functionality to options list {kibana-pull}143762[#143762]
+* Adds `excludes` toggle to options list {kibana-pull}142780[#142780]
+* Adds support for IP field to options list {kibana-pull}142507[#142507]
+
+Discover::
+* Adds the ability to edit ad-hoc data views without permissions {kibana-pull}142723[#142723]
```

Comment:
just wondering if we should label it `ad-hoc` or `adhoc`? Both is used here

Reply:
Maybe `ad hoc`? I feel like that's the most typical usage.